lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -102,6 +112,87 @@ class ReportUIFeatures {
     dropdown.addEventListener('click', this.onExport);
   }
 
+  _setupThirdPartyFilter() {
+    const thirdPartyFilterAuditExclusions = ['uses-rel-preconnect'];
```

Comment:
need a comment for what this is

Reply:
```
List of audits that shouldn't display the 3p filter feature
``` 

That? That's what I'm getting just from the name - which makes me think a comment isn't needed. @wardpeet  is it more nuanced than that?

Reply:
how bout `Some audits should not display the third party filter option.`

Reply:
I imagine @brendankenny wanted the *why* in a comment. i.e. this audit is explicitly recommending to you, only third party origins that you should be connecting to so filtering out third parties doesn't make sense

Reply:
@brendankenny all good?